File: Service/Models/Requests/PlaceWagerRequestModel.cs
using TecmoTourney;

namespace TecmoTourney.Models.Requests
{
    public class PlaceWagerRequestModel
    {
        public int GameResultId { get; set; }
        public WagerMarketType MarketType { get; set; }
        public WagerSide Side { get; set; }
        public decimal StakeAmount { get; set; }
    }
}

File: Client/tecmo-wager/src/app/core/models/place-wager-request.model.ts
export type WagerMarketType = 'Spread' | 'OverUnder' | 'MoneyLine';
export type WagerSide = 'Player1Spread' | 'Player2Spread' | 'Over' | 'Under' | 'Player1ML' | 'Player2ML';

export interface PlaceWagerRequest {
  gameResultId: number;
  marketType: WagerMarketType;
  side: WagerSide;
  stakeAmount: number;
}
